doc/examples/rendering/wedgeTests.gfr

```diff
@@ -413,61 +412,65 @@ __children["Outputs"]["outputs"]["output1"]["fileName"].setValue( '${project:roo
-__children["Backdrop12"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 7.62612152, -7.58981323 ), imath.V2f( 64.7146454, 6.37140656 ) ) )
+__children["Backdrop12"]["description"].setValue( "Now that we have renders with burn-in, there's no need to keep the original beauty renders. We can use a SystemCommand task node to issue an `rm` command to delete them, right after the copies with burn-in are created." )
+__children["Backdrop12"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 7.62612152, -8.83 ), imath.V2f( 64.7146454, 6.37140656 ) ) )
 __children["Backdrop12"]["__uiPosition"].setValue( imath.V2f( -40.4838257, -133.045593 ) )
-Gaffer.Metadata.registerValue( __children["Backdrop13"], 'nodeGadget:color', imath.Color3f( 0.368000001, 0.400000006, 0.359999985 ) )
-__children["Backdrop13"]["title"].setValue( 'Render the Wedge' )
-__children["Backdrop13"]["description"].setValue( "Right-click on the MetallicWedge node and click 'Execute' to render  the  ramp." )
-__children["Backdrop13"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 5.62234879, -3.33480835 ), imath.V2f( 62.358757, 10.0430069 ) ) )
+__children["Backdrop13"]["title"].setValue( 'Render the wedges' )
+__children["Backdrop13"]["description"].setValue( "To render the wedges in sequence, right-click the Wedge node, then select Execute from the context menu. This will trigger all the upstream task nodes to execute, in order.\n\nOnce rendered, you can preview the final images by selecting the ImageReader1 node, then scrubbing the timeline from frames 0-5." )
+__children["Backdrop13"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 5.62234879, -8.2 ), imath.V2f( 62.358757, 10.0430069 ) ) )
 __children["Backdrop13"]["__uiPosition"].setValue( imath.V2f( -37.994175, -155.492065 ) )
 __children["as_disney_material"]["parameters"]["in_metallic_amount"].setInput( __children["Expression1"]["__out"]["p0"] )
 __children["as_disney_material"]["__uiPosition"].setValue( imath.V2f( -4.3152895, 2.59905791 ) )
 __children["as_closure2surface"]["parameters"]["in_input"].setInput( __children["as_disney_material"]["out"]["out_outColor"] )
 __children["as_closure2surface"]["__uiPosition"].setValue( imath.V2f( 16.4932938, 3.79905796 ) )
 __children["Expression1"]["__engine"].setValue( 'OSL' )
 __children["Expression1"]["__expression"].setValue( 'parent.__out.p0 = context("wedge:metallic", 0.0)' )
+__children["Backdrop14"] = Gaffer.Backdrop( "Backdrop14" )
+parent.addChild( __children["Backdrop14"] )
+__children["Backdrop14"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Backdrop14"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+Gaffer.Metadata.registerValue( __children["Backdrop14"], 'nodeGadget:color', imath.Color3f( 0.36, 0.5041, 0.6 ) )
```

Comment:
You might find this a little too saturated, but I wanted to keep its color close to the one in docs, but also distinct from the Note color. Let me know.